lib/Perl/Critic/Policy/Subroutines/ProhibitUnusedPrivateSubroutines.pm

```diff
@@ -347,6 +358,13 @@ These are added to the default list of exemptions from this policy. So the
 above allows C<< sub _bar {} >> and C<< sub _baz {} >>, even if they are not
 referred to in the module that defines them.
 
+You can configure this policy not to check private subroutines declared in a
+file that uses one or more particular named classes.  This allows you to, for
+example, exclude unused private subroutine checking in classes that are roles.
+
+    [Subroutines::ProhibitUnusedPrivateSubroutines]
+    skip_when_using = Moose::Role Moo::Role Role::Tiny
+
```

Comment:
Please mention the caveats about multiple packages in the same file here.  Thanks.